features/bevragen/persoon/verblijfplaats/adres/in-onderzoek.feature

```diff
@@ -0,0 +1,189 @@
+    | 081000                  |         | true        |                          |                                  |                                     | true            | hele groep adreshouding       |
+    | 081010                  |         |             |                          |                                  |                                     | true            |                               |
+    | 081030                  |         | true        |                          |                                  |                                     |                 | datum aanvang adreshouding    |
+    | 081100                  | true    |             |                          | true                             | true                                |                 |                               |
+    | 081180                  |         |             |                          |                                  | true                                |                 |                               |
+    | 081190                  |         |             |                          | true                             |                                     |                 |                               |
+    | 088500                  |         |             | true                     |                                  |                                     |                 |                               |
+    | 088510                  |         |             | true                     |                                  |                                     |                 |                               |
+
+  Abstract Scenario: '<type>' is in onderzoek, maar 'datumVan' wordt niet gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'verblijfplaats' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                 | waarde |
+    | gemeentecode (92.10) | 0519   |
+    | straatnaam (11.10)   | Spui   |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                               |
+    | type                | RaadpleegMetBurgerservicenummer      |
+    | burgerservicenummer | 000000152                            |
+    | fields              | verblijfplaats.datumIngangGeldigheid |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam | waarde |
+    | type | Adres  |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | type                       |
+    | 081030                  | datum aanvang adreshouding |
```

Comment:
waarom doe je hier niet dezelfde dekking als in vorige scenario? Dus voor elk veld controleren dat in onderzoek niet geleverd wordt wanneer het wel in onderzoek staat maar niet gevraagd is?

En voor andere velden van categorie 8, zoals 080820 (zit buiten groep verblijfplaats), 081020 (zit niet in onze resource) of 081120 (wordt niet gevraagd)

Reply:
goed idee. omgezet